docs/reference/components/sinks/datadog_logs.cue

```diff
@@ -3,6 +3,13 @@ package metadata
 components: sinks: datadog_logs: {
 	title: "Datadog Logs"
 
+	description: """
+		Sends data to the [Datadog logs service](\(urls.datadog_logs)). The Datadog platform can leverage specifics fields (namely `ddtags`, `ddsource` and `service`) to further categorize
+		and filter logs. For further details on the purpose of those fields please check the [official Datadog documentation](\(urls.datadog_tags)).
```

Comment:
For further details on the purpose of those fields, refer to the official Datadog documentation